kustomize/README.md

```diff
@@ -44,3 +44,4 @@ oc label namespace $YOUR_NAMESPACE_NAME application=agent-morpheus
 # Deploy agent-morpheus integrated with our self-hosted LLM
 oc kustomize overlays/our-llm-service |  oc apply -f - -n $YOUR_NAMESPACE_NAME
 ```
+7. Deploying self-hosted LLM Instructions can be found [here](https://github.com/zvigrinberg/nim-deploy/blob/main/helm/README.md#launching-a-nim-llama31-8b-instruct-including-toleration-to-schedule-on-a100-node-on-openshift)
```

Comment:
These instructions should be part of this repository. Please create a dedicated README and include all necessary files in this repository so that we can maintain it and share it.

Reply:
@ruromero We can do it
But do you really want to take all the chart files to this repo?
IMO the current solution is much more elegant and decoupled.. but i can do it , no problem.
the cons of that is that we'll need to track changes in nim-deploy upstream, and via my fork, it's much more easier and straight forward...

Reply:
I mean the readme that you're pointing to your own repository.

Reply:
OK NP

Reply:
addressed as agreed